.github/workflows/pr_test.yml

```diff
@@ -197,9 +221,10 @@ jobs:
         run: npm install
         working-directory: '${{ env.PS_DIR }}/tests/UI'
 
-      - name: Run tests
+      - name: Run tests 8.0
+        if: ${{ env.BASE_BRANCH != 'develop' }}
```

Comment:
Why don't we want to run 8.0 tests for develop branch ?

Reply:
Modifications to package.json for UI Tests are only applied on `develop` branch.